server/src/test/java/com/linecorp/centraldogma/server/internal/admin/authentication/LoginAndLogoutTest.java

```diff
@@ -50,7 +56,29 @@ static AggregatedHttpMessage login(CentralDogmaRule rule, String username, Strin
                 HttpHeaders.of(HttpHeaderNames.METHOD, "POST",
                                HttpHeaderNames.PATH, "/api/v0/authenticate",
                                HttpHeaderNames.CONTENT_TYPE, MediaType.FORM_DATA.toString()),
-                "username=" + username + "&password=" + password + "&remember_me=true",
+                "grant_type=password&username=" + username + "&password=" + password,
+                StandardCharsets.US_ASCII).aggregate().join();
+    }
+
+    static AggregatedHttpMessage loginWithBasicAuth(
+            CentralDogmaRule rule, String username, String password) {
+        return rule.httpClient().execute(
+                HttpHeaders.of(HttpHeaderNames.METHOD, "POST",
+                               HttpHeaderNames.PATH, "/api/v0/authenticate",
+                               HttpHeaderNames.AUTHORIZATION, "basic " + encoder.encodeToString(
+                                       (USERNAME + ':' + PASSWORD).getBytes(StandardCharsets.US_ASCII)),
+                               HttpHeaderNames.CONTENT_TYPE, MediaType.FORM_DATA.toString()),
+                "grant_type=client_credentials",
```

Comment:
Shouldn't this be `password`?

Reply:
From the Golang oauth2 client, I thought it was `"grant_type=client_credentials"`.
But I checked again the [spec](https://tools.ietf.org/html/rfc7617#section-2.1), and it actually does not require the `grant_type`.
So I will remove it :-)

Reply:
I thought I deleted this review comment but seems like I forgot. :-$